### ISSUE TITLE
credential issue

### ISSUE THREAD

**Issue by laurelandhardy** [stormshadow07/BeeF-Over-Wan#44]
Access The BeeF Control Panel Using : http://http://45cb1c5c2fac.ngrok.io/ui/panel
	 Username = beef
	 Password = beef  (not working)

**Comment by Yaqoobi1998** [comment]
> Username = beef
> Password = beef (not working)

did you solve this problem bro? please help me

**Comment by Warhammar** [comment]
Have the same issue and no it is not admin.. have tried so many.. who did change the code...

**Comment by D4rkF4iry** [comment]
if u go to if u do $ cat /usr/share/beef-xss/config.yml u will see password there and yes the password for beef over wan aswell they same

**Comment by Aarav5255** [comment]
I don't think that that's correct. i checked the config file and used those credentials, but still it is not working.
